azuread/resource_application.go

```diff
@@ -202,7 +202,61 @@ func resourceApplication() *schema.Resource {
 				Computed: true,
 			},
 
-			"oauth2_permissions": graph.SchemaOauth2PermissionsComputed(),
+			"oauth2_permissions": {
+				Type:     schema.TypeSet,
+				Optional: true,
```

Comment:
Per [katbyte's comment](https://github.com/terraform-providers/terraform-provider-azuread/issues/162#issuecomment-543807695) and the other discussion in #162, should this also be computed?